documentation/related_projects/index.markdown

```diff
@@ -11,69 +11,69 @@ title: Related Projects
 
 MoveIt is designed to be easily extensible as separate plugins or projects. Below is a list of related projects that have been aligned with the MoveIt framework, in alphabetical order of project names. This is by no means exhaustive. Please create pull-requests in the repository [moveit.ros.org](https://github.com/ros-planning/moveit.ros.org) to **add** other projects to this list. Or if not sure about the relevance, please post on the [ROS Discourse category](https://discourse.ros.org/c/moveit).
 
-## motoman_apps
+## ROS Control
 
-The [Nishida Takeshi Lab](http://lab.cntl.kyutech.ac.jp/) in Fukuoka, Japan supports a suite of ROS packages for working with the Motoman line of Robots from [Yaskawa](https://www.motoman.com/) through a number of interfaces including Matlab. You can explore some of these interfaces in the [motoman_apps](https://github.com/ntl-ros-pkg/motoman_apps) repository.
+A set of packages that include controller interfaces, controller managers, transmissions and hardware_interfaces. The ros_control packages are a rewrite of the [pr2_mechanism](http://wiki.ros.org/pr2_mechanism) packages to make controllers generic to all robots beyond just the PR2. The ros_control packages takes as input the joint state data from your robot's actuator's encoders and an input set point. It uses a generic control loop feedback mechanism, typically a PID controller, to control the output, typically effort, sent to your actuators. ros_control gets more complicated for physical mechanisms that do not have one-to-one mappings of joint positions, efforts, etc but these scenarios are accounted for using transmissions.
 
-<img src="/assets/images/related_projects/motoman_apps.png" height="500"/>
+<img src="/assets/images/related_projects/ros_control.png" height="500"/>
 
-## bio_ik
+## TracIK
+
+Provide an alternative Inverse Kinematics solver to the popular inverse Jacobian methods in KDL. Specifically, KDL's convergence algorithms are based on Newton's method, which does not work well in the presence of joint limits --- common for many robotic platforms. [More Info](https://bitbucket.org/traclabs/trac_ik)
+
+<img src="/assets/images/related_projects/place_holder.png" height="300"/>
+
+## BioIK
```

Comment:
The module is named `bio_ik`. BioIK is actually a specific algorithm with a different inventor, so the name must not change for political reasons.

Reply:
Changed to Bio_IK. I think headings should be capitalized, though. Nicer appearance for a webpage

Reply:
Sorry, but that's not enough for me. If you want the heading capitalized, please add a description like 
```
Generic IK Plugin `bio_ik`
```